src/components/InterestForm/styles.tsx

```diff
@@ -0,0 +1,66 @@
+import styled from 'styled-components';
+import COLORS from '../../styles/colors';
+
+export const FormContainer = styled.form`
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-end;
+  width: 60%;
+  float: right;
+  margin-top: 1rem;
+  gap: 0.5rem;
+`;
+
+export const FormTextArea = styled.textarea`
+  height: 16vh;
+  resize: none;
+  border-radius: 5px;
+  border: 1px solid var(--mid-grey, #ccc);
```

Comment:
I think we want to reference the `COLORS` object instead of using CSS variables, but we should discuss this!

Reply:
Fixed! Just haven't pushed the changes yet